File: TestApplication.Services/Models/ImageUploadModel.cs
using Microsoft.AspNetCore.Http;
using System;
using System.Collections.Generic;
using System.Text;

namespace TestApplication.Services.Models
{
    public class ImageUploadModel
    {
        public string Name { get; set; }
        public string Nic { get; set; }
        public IFormFile NicCopy { get; set; }
        public IFormFile ProfilePic { get; set; }
    }
}

File: TestApplication.WEB/ClientApp/src/app/models/ImageUploadModel.ts
export interface FileParameter {
    data: any;
    fileName: string;
  }
  
  export class ImageUploadModel{
    name: string
    nic: string
    nicCopy: FileParameter
    profilePic: FileParameter
  }

  export class UploadResultModel{
    url: string
  }